ui/src/i18n/en.json

```diff
@@ -309,9 +309,12 @@
   "settings.service.debug_tooltip": "Some plugins require you to set the DEBUG environment variable to enable debug level logging.",
   "settings.service.node_tooltip": "Node.js runtime options for this process. This should be kept blank unless you know what you are doing.",
   "settings.startup.debug": "Homebridge Debug Mode",
+  "settings.startup.debug_help": "Enable debug logs for system and plugins.",
   "settings.startup.insecure_1": "Homebridge 'Insecure' Mode",
   "settings.startup.insecure_2": "Enable Accessory Control",
+  "settings.startup.insecure_help": "To enable accessory control directly in Homebridge (Accessory page) Insecure Mode must be enabled. Warning: Anyone with network access to your Homebridge server port (i.e your local network, unless you expose port to the internet) and pin will be able to control your accessories.",
   "settings.startup.keep_accessories": "Keep Accessories Of Uninstalled Platform Plugins",
+  "settings.startup.keep_accessories_help": "When this option is enabled, after removing the plugin, the accessories will stay in the cache. To delete them you have to do it manually in Homebridge Settings. In many cases, this option also retains accessories in case the plugin does not run properly.",
```

Comment:
> In many cases, this option also retains accessories in case the plugin does not run properly.

in which cases?

Reply:
Mainly when, as a result of an error, the plugin gives a signal as if it has no devices, but it does, but it did not start properly.

Reply:
The problem may even be the lack of a comma. When the problem is solved, the plugin returns to normal. But if I didn't have the 'keep accessories' option selected it would remove my accessories because of this error and after fixing the problem it would add these accessories as new and I would have to set them again in Apple Home.

Reply:
Maybe it also means that as a result of an error the plugin will not be published correctly? And because of this, the system thinks plugin is uninstalled?

Reply:
This options should be named : Remove plugin accessories from cache only manually.

Reply:
Or at least "Keep accessories of plugins"